tools/mo/openvino/tools/mo/utils/find_ie_version.py

```diff
@@ -94,7 +94,7 @@ def find_ie_version(silent=False):
     bindings_paths_windows = [
         # Local builds
         {
-            "module": os.path.join(script_path, '../../../../../../bin/intel64/Release/python_api/', python_version),
+            "module": os.path.join(script_path, '../../../../../../bin/intel64/Release/Release/python_api/', python_version),
```

Comment:
Is it really that path on Windows? This doesn't seem to be in scope of this PR though

Reply:
not present anymore